internal/app/singularity/capability.go

```diff
@@ -3,7 +3,7 @@
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-// +build singularity_runtime
+// +build linux
```

Comment:
Would it be better to use file selection here (rename to `capability_linux.go`) here and in the other ~10 files with `+build linux`?